gp_theta:
  group_size: 26
  overlap_size: 6
  boundary_margin: 0.017500000000000002
  init_variance: 1000000
  sensor_range_var: 0.01
  max_valid_distance_var: 0.10000000000000001
  occ_test_temperature: 30
  train_buffer:
    valid_range_min: 0.20000000000000001
    valid_range_max: 30
    valid_angle_min: -2.3561944901923448
    valid_angle_max: 2.3561944901923448
    mapping:
      type: kInverseSqrt
      scale: 1
  gp:
    kernel:
      type: kOrnsteinUhlenbeck
      alpha: 1
      scale: 0.5
      scale_mix: 1
      weights:
        []
    auto_normalize: false
compute_variance:
  zero_gradient_position_var: 1
  zero_gradient_gradient_var: 1
  min_distance_var: 1
  max_distance_var: 100
  position_var_alpha: 0.01
  min_gradient_var: 0.01
  max_gradient_var: 1
quadtree_resolution: 0.1
cluster_level: 3
perturb_delta: 0.01
update_occupancy: false